Header header

string LANE_POST = "/processing/roadline_post_processing"
string STOP_DETECT = "/processing/stopline_detection_output"
string ENV_OUTPUT = "/processing/environment_prediction"
string HLDF_OUTPUT = "/processing/environment_hldf"
string HD_MAP = "/processing/hd_map"
